.github/workflows/deleteOldReleases.yml

```
@@ -0,0 +1,51 @@
+name: Delete Old Releases
+
+on:
+  schedule:
+    - cron: '0 10 * * *'  # Daily at 10:00 UTC which is 2:00 AM PST (UTC-8)
+  workflow_dispatch:      # Manual trigger
+  push: 
```

I would actually skip this trigger.

```suggestion
  push:
    branches:
      - main
```

---

To confirm, you are suggesting to remove the `push` trigger or add the `branches main` to the trigger?

I originally added it to trigger the CI from my fork but forgot to remove it.

---

> you are suggesting to remove the `push` trigger or add the `branches main` to the trigger?

Sorry for the confusing comment - I suggest either of those with a preference for the former :)